From 6d4293163fd33a15c529eaae70b636f23b04e21c Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Mon, 8 Apr 2019 14:51:47 +0800
Subject: [PATCH 13/30] gpu/imx/dcss: include linux/seq_file.h where needed

The dcss-*.c need include this header implicitly to avoid
compile error as below:

error: implicit declaration of function 'seq_printf' [-Werror=implicit-function-declaration]

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpu/imx/dcss/dcss-blkctl.c | 1 +
 drivers/gpu/imx/dcss/dcss-ctxld.c  | 1 +
 drivers/gpu/imx/dcss/dcss-dtg.c    | 1 +
 drivers/gpu/imx/dcss/dcss-dtrc.c   | 1 +
 drivers/gpu/imx/dcss/dcss-hdr10.c  | 1 +
 drivers/gpu/imx/dcss/dcss-rdsrc.c  | 1 +
 drivers/gpu/imx/dcss/dcss-scaler.c | 1 +
 drivers/gpu/imx/dcss/dcss-ss.c     | 1 +
 drivers/gpu/imx/dcss/dcss-wrscl.c  | 1 +
 9 files changed, 9 insertions(+)

diff --git a/drivers/gpu/imx/dcss/dcss-blkctl.c b/drivers/gpu/imx/dcss/dcss-blkctl.c
index 2f13b33dba35..5123eb995790 100644
--- a/drivers/gpu/imx/dcss/dcss-blkctl.c
+++ b/drivers/gpu/imx/dcss/dcss-blkctl.c
@@ -18,6 +18,7 @@
 #include <linux/of.h>
 #include <linux/delay.h>
 #include <soc/imx8/soc.h>
+#include <linux/seq_file.h>
 
 #include "dcss-prv.h"
 #include <video/imx-dcss.h>
diff --git a/drivers/gpu/imx/dcss/dcss-ctxld.c b/drivers/gpu/imx/dcss/dcss-ctxld.c
index 377a102ad55f..993616beffa1 100644
--- a/drivers/gpu/imx/dcss/dcss-ctxld.c
+++ b/drivers/gpu/imx/dcss/dcss-ctxld.c
@@ -21,6 +21,7 @@
 #include <linux/dma-mapping.h>
 #include <linux/delay.h>
 #include <asm/cacheflush.h>
+#include <linux/seq_file.h>
 
 #include "video/imx-dcss.h"
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-dtg.c b/drivers/gpu/imx/dcss/dcss-dtg.c
index 47f43cc426c7..df091e766a8d 100644
--- a/drivers/gpu/imx/dcss/dcss-dtg.c
+++ b/drivers/gpu/imx/dcss/dcss-dtg.c
@@ -21,6 +21,7 @@
 #include <linux/platform_device.h>
 #include <linux/interrupt.h>
 #include <drm/drm_fourcc.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-dtrc.c b/drivers/gpu/imx/dcss/dcss-dtrc.c
index 6245e2bd883a..afac5a3d80b7 100644
--- a/drivers/gpu/imx/dcss/dcss-dtrc.c
+++ b/drivers/gpu/imx/dcss/dcss-dtrc.c
@@ -19,6 +19,7 @@
 #include <linux/interrupt.h>
 #include <drm/drm_fourcc.h>
 #include <linux/delay.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-hdr10.c b/drivers/gpu/imx/dcss/dcss-hdr10.c
index 7dad29acf2e1..1e3b47431ca5 100644
--- a/drivers/gpu/imx/dcss/dcss-hdr10.c
+++ b/drivers/gpu/imx/dcss/dcss-hdr10.c
@@ -17,6 +17,7 @@
 #include <linux/io.h>
 #include <linux/firmware.h>
 #include <drm/drm_fourcc.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-rdsrc.c b/drivers/gpu/imx/dcss/dcss-rdsrc.c
index 1fa9c099dfc7..d011e33bd894 100644
--- a/drivers/gpu/imx/dcss/dcss-rdsrc.c
+++ b/drivers/gpu/imx/dcss/dcss-rdsrc.c
@@ -15,6 +15,7 @@
 #include <linux/device.h>
 #include <linux/io.h>
 #include <linux/dma-mapping.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-scaler.c b/drivers/gpu/imx/dcss/dcss-scaler.c
index abddd129169c..e44ec4758933 100644
--- a/drivers/gpu/imx/dcss/dcss-scaler.c
+++ b/drivers/gpu/imx/dcss/dcss-scaler.c
@@ -16,6 +16,7 @@
 #include <linux/platform_device.h>
 #include <linux/io.h>
 #include <drm/drm_fourcc.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-ss.c b/drivers/gpu/imx/dcss/dcss-ss.c
index 66df49e565df..ac759b7deec2 100644
--- a/drivers/gpu/imx/dcss/dcss-ss.c
+++ b/drivers/gpu/imx/dcss/dcss-ss.c
@@ -17,6 +17,7 @@
 #include <linux/io.h>
 #include <video/videomode.h>
 #include <drm/drm_fourcc.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
diff --git a/drivers/gpu/imx/dcss/dcss-wrscl.c b/drivers/gpu/imx/dcss/dcss-wrscl.c
index 30b225958cf7..8a285979e431 100644
--- a/drivers/gpu/imx/dcss/dcss-wrscl.c
+++ b/drivers/gpu/imx/dcss/dcss-wrscl.c
@@ -16,6 +16,7 @@
 #include <linux/io.h>
 #include <linux/clk.h>
 #include <linux/dma-mapping.h>
+#include <linux/seq_file.h>
 
 #include <video/imx-dcss.h>
 #include "dcss-prv.h"
-- 
2.17.1

